lib/active_record_doctor/detectors/missing_non_null_constraint.rb

```diff
@@ -18,8 +18,14 @@ class MissingNonNullConstraint < Base # :nodoc:
 
       private
 
+      TIMESTAMPS = ["created_at", "created_on", "updated_at", "updated_on"].freeze
+
       def message(column:, table:)
-        "add `NOT NULL` to #{table}.#{column} - models validates its presence but it's not non-NULL in the database"
+        if TIMESTAMPS.include?(column)
+          "add `NOT NULL` to #{table}.#{column}"
```

Comment:
We should also explain to the user why we're making this recommendation. In this case, I suggest something like:

```
timestamp columns are set automatically by Active Record and allowing NULL may lead to inconsistencies introduced by bulk operations
```

Reply:
Thanks for the review and suggestion. I applied this in the latest revision.

Reply:
Thank you!